helix-core/src/main/java/org/apache/helix/HelixAdmin.java

```diff
@@ -380,6 +380,13 @@ void addStateModelDef(String clusterName, String stateModelDef, StateModelDefini
    */
   void dropResource(String clusterName, String resourceName);
 
+  /**
+   *
+   * @param clusterName
+   * @param cloudConfig
+   */
+  void addCloudConfig(String clusterName, CloudConfig cloudConfig);
```

Comment:
So we never remove the config?
For a complete operation set, I'd prefer we have a delete cluster config interface. Or call it setCloudConfig so it can be removed by passing null parameter.

Reply:
Ok removeCloudConfig is added.